pyrasterframes/python/pyrasterframes/rasterfunctions.py

```diff
@@ -116,137 +116,133 @@ def _(geometryCol, srcCRSName, dstCRSName):
+    'rf_local_greater_double': 'Return a Tile with values equal 1 if the cell is greater than a scalar, otherwise 0',
+    'rf_local_greater_int': 'Return a Tile with values equal 1 if the cell is greater than a scalar, otherwise 0',
+    'rf_local_greater_equal_double': 'Return a Tile with values equal 1 if the cell is greater than or equal to a scalar, otherwise 0',
+    'rf_local_greater_equal_int': 'Return a Tile with values equal 1 if the cell is greater than or equal to a scalar, otherwise 0',
+    'rf_local_equal_double': 'Return a Tile with values equal 1 if the cell is equal to a scalar, otherwise 0',
+    'rf_local_equal_int': 'Return a Tile with values equal 1 if the cell is equal to a scalar, otherwise 0',
+    'rf_local_unequal_double': 'Return a Tile with values equal 1 if the cell is not equal to a scalar, otherwise 0',
+    'rf_local_unequal_int': 'Return a Tile with values equal 1 if the cell is not equal to a scalar, otherwise 0',
 }
 
 
 _rf_column_functions = {
     # ------- RasterFrames functions -------
-    'tile_dimensions': 'Query the number of (cols, rows) in a Tile.',
-    'envelope': 'Extracts the bounding box (envelope) of the geometry.',
-    'tile_to_int_array': 'Flattens Tile into an array of integers. Deprecated in favor of `tile_to_array_int`.',
-    'tile_to_double_array': 'Flattens Tile into an array of doubles. Deprecated in favor of `tile_to_array_double`',
-    'tile_to_array_int': 'Flattens Tile into an array of integers.',
-    'tile_to_array_double': 'Flattens Tile into an array of doubles.',
-    'cell_type': 'Extract the Tile\'s cell type',
-    'is_no_data_tile': 'Report if the Tile is entirely NODDATA cells',
-    'agg_approx_histogram': 'Compute the full column aggregate floating point histogram',
-    'agg_stats': 'Compute the full column aggregate floating point statistics',
-    'agg_mean': 'Computes the column aggregate mean',
-    'agg_data_cells': 'Computes the number of non-NoData cells in a column',
-    'agg_no_data_cells': 'Computes the number of NoData cells in a column',
-    'tile_histogram': 'Compute the Tile-wise histogram',
-    'tile_mean': 'Compute the Tile-wise mean',
-    'tile_sum': 'Compute the Tile-wise sum',
-    'tile_min': 'Compute the Tile-wise minimum',
-    'tile_max': 'Compute the Tile-wise maximum',
-    'tile_stats': 'Compute the Tile-wise floating point statistics',
-    'render_ascii': 'Render ASCII art of tile',
-    'no_data_cells': 'Count of NODATA cells',
-    'data_cells': 'Count of cells with valid data',
-    'local_add': 'Add two Tiles',
-    'local_subtract': 'Subtract two Tiles',
-    'local_multiply': 'Multiply two Tiles',
-    'local_divide': 'Divide two Tiles',
-    'normalized_difference': 'Compute the normalized difference of two tiles',
-    'local_agg_stats': 'Compute cell-local aggregate descriptive statistics for a column of Tiles.',
```

Comment:
looks like we inadvertently dropped `rf_agg_local_stats` from the dict. Need to add back in

Reply:
Maybe worth trying to create a test that just tries to use all of these keys, that would at least make sure the keys are not mis-spelled.

Reply:
Added it back in.

I added the file `pyrasterframes/python/coverage-report.sh` To run tests with coverage tracking on. Helps a tiny bit, but due to the dynamic trickery going on, doesn't track to the level necessary. We should soon rethink how the `PyRasterFramesTests` is structured and what we might to to make it easier to ensure fuller coverage.

Reply:
Indeed. Also because of the dynamic trickery we loose functionality with IDE's and Jupyter code completion.